gms/gossiper.cc

```diff
@@ -1761,38 +1761,83 @@ future<> gossiper::start_gossiping(int generation_nbr, std::map<application_stat
     });
 }
```

Comment:
Please also test upgrades and downgrades from pre-seedless.

Reply:
@roydahan QA should prepare for this.

Reply:
I agree, let's run dtest BYO even before merging this because it can break even next-gating tests.
We will start making sure that SCT will be ready for it.

Reply:
I started a byo test but the test failed:

```
Error Message

unsupported url or file doesn't exist
	url=/jenkins/workspace/scylla-master/byo/byo_build_tests_dtest/scylla-tools-java/build/scylla-tools-package.tar.gz
```
E.g., 

https://jenkins.scylladb.com/job/scylla-master/job/byo/job/byo_build_tests_dtest/986/testReport/junit/rebuild_test/TestRebuild/dtest_not_Parallel___simple_rebuild_test/

I changed the scylla git url and branch to test. All others are default. What else should I configure to make the byo test run?